dynatrace-aws-logs.sh

```diff
@@ -56,7 +56,7 @@ case $MAIN_OPTION in
 
   function print_help_deploy {
     printf "
-usage: dynatrace-aws-logs.sh deploy --target-url TARGET_URL --target-api-token TARGET_API_TOKEN --use-existing-active-gate {true|false} [--target-paas-token TARGET_PAAS_TOKEN] [--require-valid-certificate {true|false}] [--stack-name STACK_NAME]
+usage: dynatrace-aws-logs.sh deploy --target-url TARGET_URL --target-api-token TARGET_API_TOKEN [--use-existing-active-gate {true|false}] [--target-paas-token TARGET_PAAS_TOKEN] [--require-valid-certificate {true|false}] [--stack-name STACK_NAME]
 
 arguments:
   -h, --help            show this help message and exit
```

Comment:
Can't comment the correct line - in 73 we should probably mention that it defaults

Reply:
I kept it like that because in the Azure forwarder the optional usage is only indicated in that first line of the help, with those brackets, and enough with it. There were already optional arguments that have no explicit "Optional" written. But never mind, minor change, so I can add it.

Reply:
As far as I can see if there is no "Optional" written then it says something about "by default" or "this defaults to". Yeah, it is minor though.